docs/operator-manual/argocd-cm.yaml

```diff
@@ -216,6 +216,9 @@ data:
   kustomize.version.v3.5.1: /custom-tools/kustomize_3_5_1
   kustomize.version.v3.5.4: /custom-tools/kustomize_3_5_4
 
+  # Comma delimited list of additional custom remote values file schemes (http are https are allowed by default)
+  helm.valuesFileSchemes: s3, git
```

Comment:
Would `helm.valuesFileSchemesEnabled` or ``helm.valuesFileSchemesAllowed`` be better?

Reply:
For some reason, I like just `helm.valuesFileSchemes`.  Boh `enabled` and `allowed` looks a little redundant to me because I cannot imagine any other option. 

I might be wrong and don't have a strong opinion about it, to be honest.  @terrytangyuan let me know please if you really think that some clarification is required and we should use `helm.valuesFileSchemesEnabled`/`helm.valuesFileSchemesAllowed`.

Reply:
Sounds good to me. No need to change.